.circleci/requirements.txt

```diff
@@ -1,3 +1,4 @@
 flake8==3.7.9
+pandasql==0.7.3
```

Comment:
I can't explain why this is needed here but other dependent packages are apparently not? I need to do more research but for now this dependency is needed to run tests in circle ci.

Reply:
I'll look into this during my CircleCI research. I'm interested to see what happens using pip and the setup.py file alone.